include/ListView/ListViewSubPanel.php

```diff
@@ -795,6 +795,21 @@ function mailmerge_dialog(el) {
                         $subpanelTiles = new SubPanelTiles($sugarbean);
                         $html_text .= "<table border=\"0\" cellpadding=\"0\" cellspacing=\"0\" width=\"100%\"><tr><td align=\"left\"  >";
 
+                        if (isset($subpanelTiles->subpanel_definitions->layout_defs['subpanel_setup']['delegates']['select_link_top']) &&
+                            $subpanelTiles->subpanel_definitions->layout_defs['subpanel_setup']['delegates']['select_link_top'] === true) {
+                            $html_text .= '<ul id="selectLinkTop" class="clickMenu selectmenu SugarActionMenu" name="">
```

Comment:
It should works but I think better solution separate this code into a new method of SubPanelTiles class to make it callable anywhere and the html view into a smarty template and generate from .tpl and remove the inline 'style' attributes and adding to themes (css/scss)